src/components/landing-page/TalentSection/TalentSection.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import styles from './TalentSection.module.scss'
+import FindTalentsBtn from '@/components/FindTalentsBtn/FindTalentsBtn'
+import ProfileCard from '@/components/ProfileCard/ProfileCard'
+import { getPublishedProfilesPayload } from '@/backend/profile/profile.service'
+import Link from 'next/link'
+import { AppRoutes } from '@/utils/routes'
+import { ProfileModel } from '@/data/frontend/profile/types'
+
+const TalentSection = async () => {
+  const profiles = await getPublishedProfilesPayload()
+
+  let randomProfiles: ProfileModel[] = []
```

Comment:
**issue:** We should not do it this way - we should get random 6 records from DB instead of getting all of them, loading to memory, and filter on FE. Let's create new method eg: `getRandomProfiles(amount: number)`, and take only 4 records from DB.